mgmt_cli_test.py

```diff
@@ -258,19 +258,45 @@ def run_verification_read_stress(self, ks_names=None):
+    def prepare_run_and_verify_stress_in_threads(self, cmd_template: str, keyspace_name: str, num_of_rows: int,
+                                                 stop_on_failure: bool = False) -> None:
+        """Prepares C-S commands, runs them in threads and verifies their execution results.
+        Stress operation can be either read or write, depending on the cmd_template.
+        """
         stress_queue = []
-        number_of_loaders = self.params.get("n_loaders")
-        rows_per_loader = int(number_of_rows / number_of_loaders)
-        for loader_index in range(number_of_loaders):
-            stress_command = command_template.format(num_of_rows=rows_per_loader,
-                                                     keyspace_name=keyspace_name,
-                                                     sequence_start=rows_per_loader * loader_index + 1,
-                                                     sequence_end=rows_per_loader * (loader_index + 1),)
-            read_thread = self.run_stress_thread(stress_cmd=stress_command, round_robin=True,
-                                                 stop_test_on_failure=False)
-            stress_queue.append(read_thread)
-        return stress_queue
+        num_of_loaders = self.params.get("n_loaders")
+        rows_per_loader = int(num_of_rows / num_of_loaders)
+        for loader_index in range(num_of_loaders):
+            stress_cmd = cmd_template.format(num_of_rows=rows_per_loader,
+                                             keyspace_name=keyspace_name,
+                                             sequence_start=rows_per_loader * loader_index + 1,
+                                             sequence_end=rows_per_loader * (loader_index + 1),)
+            _thread = self.run_stress_thread(stress_cmd=stress_cmd, round_robin=True,
+                                             stop_test_on_failure=stop_on_failure)
+            stress_queue.append(_thread)
+
+        for _thread in stress_queue:
+            assert self.verify_stress_thread(cs_thread_pool=_thread), "Stress thread verification failed"
+
+    @staticmethod
+    def extract_compaction_strategy_from_cs_cmd(cs_cmd: str, lower: bool = True, remove_postfix: bool = True) -> str:
+        """Extracts the compaction strategy from the cassandra-stress command.
+
+        :param cs_cmd: cassandra-stress command
+        :param lower: if True, the resulting string will be lowercased
+        :param remove_postfix: if True, the resulting string will have the "CompactionStrategy" postfix removed
+        """
+        match = re.search(r"compaction\(strategy=([^)]+)\)", cs_cmd)
+        if match:
+            strategy = match.group(1)
+            if remove_postfix:
+                strategy = re.sub(r"CompactionStrategy$", "", strategy)
+            if lower:
+                strategy = strategy.lower()
+        else:
+            raise ValueError("Compaction strategy not found in cs_cmd.")
```

Comment:
if strategy is not provided in c-s then default one is used (size tiered for oss and incremental for enterprise)